index.html

```diff
@@ -88,7 +89,7 @@ <h1>I'm Jonathan Glad to see you</h1>
 
   <section class="project-container">
```

Comment:
- [ ] As the previous reviewer mentioned about the background-color of the sections and cards should be similar to the figma design. The origin of the problem for this is line no `47` in your style.css where you are styling your `section tag` generally. kindly consider removing that line and then give the `hero section` and the `about me` section background-color of `#fff`
your design
![re](https://user-images.githubusercontent.com/96683843/221005923-bc03bcae-bce3-4c5c-8c15-b5f6fd399242.jpg)
![fd](https://user-images.githubusercontent.com/96683843/221005939-de2578b9-7d6d-4bd8-a2f3-709439a3a987.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/221005771-1292ec61-7d2a-4bc2-804d-a5d6215e1e94.jpg)
![zx](https://user-images.githubusercontent.com/96683843/221005852-87265bec-470a-4fa0-a703-87f1a09f0ffd.jpg)